src/Application.js

```diff
@@ -19,10 +19,10 @@ class Application {
         right: {
           screen: Screens.Drawer,
           passProps: {},
-          disableOpenGesture: false,
-          fixedWidth: '20%',
+          disableOpenGesture: true,
+          fixedWidth: 600,
         },
-        disableOpenGesture: false,
+        disableOpenGesture: true,
```

Comment:
There's an issue with our version of react-native-navigation that is making this option useless. Its out of your control for now.